src/openrct2/Game.h

```diff
@@ -93,6 +93,7 @@ enum GAME_COMMAND
     GAME_COMMAND_BALLOON_PRESS,
     GAME_COMMAND_MODIFY_TILE,
     GAME_COMMAND_EDIT_SCENARIO_OPTIONS,
+    GAME_COMMAND_PLACE_PEEP_SPAWN,
```

Comment:
I just saw that the game commands that use game actions have a `// GA` here.

Reply:
Indeed. And since this was never a game command in the first place, it should not need to be specified.

Reply:
Game actions require a unique ID, and up until now the IDs from the command list could be used. Creating a new array with just game actions would be nicer indeed.